.gitignore

```diff
@@ -29,6 +29,11 @@ local.properties
 # PDT-specific
 .buildpath
 
+#################
+## IDEA
+#################
+.idea/
+*.iml
```

Comment:
Visual Studio is the standard editor for *all projects*.

I'm hesitant about encouraging people to use a rouge IDE like IntelliJ.